geonode/people/tests.py

```diff
@@ -911,3 +911,96 @@ def test_delete_a_user_with_resource(self):
         # bobby cant be deleted
         self.assertNotEqual(get_user_model().objects.filter(username="bobby").first(), None)
         self.assertTrue("user_has_resources" in response.json()["errors"][0])
+
+    def test_transfer_resources_all(self):
+        """
+        user wants to transfer resources to target
+        """
+        bobby = get_user_model().objects.get(username="bobby")
+        norman = get_user_model().objects.get(username="norman")
+        # group = GroupProfile.objects.get(slug="bar")
+
+        self.assertTrue(self.client.login(username="bobby", password="bob"))
+        self.assertTrue(bobby.is_authenticated)
+        # check bobbys resources
+        prior_bobby_resources = ResourceBase.objects.filter(owner=bobby).all()
+        self.assertTrue(len(prior_bobby_resources))
```

Comment:
you can use the `.exists()` to verify if the user has resources like: 
```python
self.assertTrue(ResourceBase.objects.filter(owner=bobby).exists())
```

No `all `+ `len `is required in this case